Logger/TrackingPropertyAttribute.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace LoggerBL
+{
+    public class TrackingPropertyAttribute : Attribute { }
```

Comment:
This attribute could be applied for any entity. Check https://docs.microsoft.com/en-us/dotnet/api/system.attributeusageattribute?view=net-6.0.

Also, it should have Name property, that will be used for by logger